bigtable/tests/data_integration_test.cc

```diff
@@ -30,66 +92,231 @@ int main(int argc, char* argv[]) try {
   std::string const project_id = argv[1];
   std::string const instance_id = argv[2];
   std::string const table_name = argv[3];
-  std::string const family = "fam";
-
-  auto admin_client =
-      bigtable::CreateDefaultAdminClient(project_id, bigtable::ClientOptions());
-  bigtable::TableAdmin admin(admin_client, instance_id);
-
-  auto created_table = admin.CreateTable(
-      table_name, bigtable::TableConfig(
-                      {{family, bigtable::GcRule::MaxNumVersions(1)}}, {}));
-  std::cout << table_name << " created successfully" << std::endl;
-
-  auto table_list = admin.ListTables(admin_proto::Table::NAME_ONLY);
-  bool found = false;
-  for (auto& tbl : table_list) {
-    if (tbl.name() == created_table.name()) {
-      found = true;
-      break;
-    }
-  }
-  if (not found) {
-    throw std::runtime_error("Could not find table test table");
-  }
-  std::cout << table_name << " found via ListTables()" << std::endl;
 
-  auto client = bigtable::CreateDefaultDataClient(project_id, instance_id,
-                                                  bigtable::ClientOptions());
-  bigtable::Table table(client, table_name);
+  (void)::testing::AddGlobalTestEnvironment(
```

Comment:
Please check that the instance is empty (i.e. `ListTables()` returns an empty list), like `filters_integration_test.cc` does.  I think we prefer a clear error message when the test starts than more obscure error messages later.

Reply:
added to main, Fixed